### PR TITLE
[AMS2020] Let's get this party started

### PR DESCRIPTION
**Pull request by thomaskrag**
We're doing another event next year. Who could have guessed?

This PR initializes the devopsdays Amsterdam 2020 event.
Created with `utilities/add_new_event.sh` and only added organizers for next year.

Let me know if anything else needs to be added.

### CODE REVIEW

File: data/events/2020-amsterdam.yml [hunk 1]
```diff
@@ -0,0 +1,105 @@
+name: "2020-amsterdam" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
+year: "2020" # The year of the event. Make sure it is in quotes.
+city: "Amsterdam" # The displayed city name of the event. Capitalize it.
+event_twitter: "dodams" # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
+description: "Hindsight is 2020!" # Edit this to suit your preferences
```
Comment (yvovandoorn) [inline data/events/2020-amsterdam.yml]: You should be explicit about it being "Coming Soon" per https://devopsdays.org/organizing/ when no venue or dates are listed.

**Comment by bridgetkromhout** [comment]
This PR removes organizers, so they need to be removed from the amsterdam email list (all four of them) and from Slack (except for those on the core team): Yvo van Doorn, Mine Heck, Wilco Noordermeer, Sarah Gruneisen.

It also adds new organizers, so it cannot be merged until they are added to Slack and to the amsterdam email list.  Floor Drees, Daniela Resch, Walter Heck. Their emails need to be sent to info@devopsdays.org.

**Comment by thomaskrag** [comment]
Hi @bridgetkromhout . We sent an email on 19 Sep, 21:04 CET from Sean Bovaird <seanbovaird@me.com> about getting the new organizers emails added to our email.

The removing from Slack I assume will be up to the core organizers.

**Comment by bridgetkromhout** [comment]
Thanks, @thomaskrag! (I was being explicit because there are new folks on the team and I want to make sure they understand how we process these.)

**Pull request merged**